cobigen-maven/cobigen-maven-systemtest/src/test/resources/testdata/systemtest/Devon4JTemplateTest/TestEntityInputDataaccessGeneration/pom.xml

```diff
@@ -37,57 +37,57 @@
         <dependency>
             <groupId>com.devonfw.cobigen</groupId>
             <artifactId>templates-devon4j</artifactId>
-            <version>3.1.8</version>
```

Comment:
íf possible keep an old version of templates as well to also test backward compatibility

Reply:
So should every test have a dependency for an old version and the new version or only the old version?